configs/sites/discover-scu17/compilers.yaml

```diff
@@ -22,6 +22,25 @@ compilers:
       set:
         # https://github.com/JCSDA/spack-stack/issues/1012
         I_MPI_EXTRA_FILESYSTEM: 'ON'
+        # https://github.com/JCSDA/spack-stack/issues/1011
+        I_MPI_SHM_HEAP_VSIZE: '512'
+        PSM2_MEMORY: 'large'
+        I_MPI_EXTRA_FILESYSTEM: '1'
+        I_MPI_EXTRA_FILESYSTEM_FORCE: 'gpfs'
+        I_MPI_FALLBACK: '0'
```

Comment:
This can probably be removed. I think it's a legacy of olden days. When you use it:
```
MPI startup(): I_MPI_FALLBACK environment variable is not supported.
```

I'll make a PR in our repo to remove as well.

Reply:
Note: I meant the `I_MPI_FALLBACK` line. Not all of those in the above GitHub dialog thing

Reply:
Done in my last commit, thanks!